flake8_django/checker.py

```diff
@@ -9,7 +9,10 @@
     RenderChecker,
 )
 
-__version__ = '0.0.4'
+__version__ = '1.1.1'
```

Comment:
hi @rocioar, wouldn't have to be 1.2.1?
and in the flake8_django/VERSION as well.

Reply:
I will make a new PR to bump the version. I was actually fixing the version number on this one.